model/converter/json/from_domain.go

```diff
@@ -151,3 +151,27 @@ func (fd fromDomain) convertProcess(process *model.Process) json.Process {
 		Tags:        fd.convertKeyValues(process.Tags),
 	}
 }
+
+// DependenciesFromDomain converts []model.DependencyLink into []json.DependencyLink format.
+// It assumes that the domain model is valid, namely that all enums
+// have valid values, so that it does not need to check for errors.
+func DependenciesFromDomain(dependencyLinks []model.DependencyLink) []json.DependencyLink {
+	return dependencyFromDomain{}.fromDomain(dependencyLinks)
+}
+
+type dependencyFromDomain struct{}
```

Comment:
this serves no purpose, the code is simple enough to be directly in the public function